gdk/src/wayland/wayland_display.rs

```diff
@@ -0,0 +1,67 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::Atom;
+use glib::translate::*;
+
+glib::wrapper! {
+    #[doc(alias = "GdkWaylandDisplay")]
+    pub struct WaylandDisplay(Object<ffi::GdkWaylandDisplay>) @extends crate::Display;
+
+    match fn {
+        type_ => || ffi::gdk_wayland_display_get_type(),
+    }
+}
+
+impl WaylandDisplay {
+    #[doc(alias = "gdk_wayland_selection_clear_targets")]
```

Comment:
where does this one comes from? i don't see it in https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/gdk/wayland/gdkwaylanddisplay.h

also lacks two wayland specific getters

Reply:
ah this one is a simple function, not a Display method iirc. same as for the other one in gdkwaylandselection.h header